tests/x11regressions/gnomecase/application_starts_on_login.pm

```diff
@@ -0,0 +1,117 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+
+#testcase 5255-1503973: Gnome: Applications starts on login 
+
+sub tweak_startupapp_menu {
+    my ($string) = @_;
+
+    wait_screen_change {
+        send_key "super";
+    };
+    type_string "settings", 1;
```

Comment:
Thank you Okurz for all your comments:)  I improved all except this one:  "1" is very slow, so I use it to give gnome-shell enough searching time for the settings module. Otherwise, slow worker will cause failed result.